harness/determined/profiler.py

```diff
@@ -78,9 +129,7 @@ def __init__(self, trial_id, agent_id, master_url, start_on_batch, end_after_bat
         self.sys_metric_collector_thread = SysMetricCollectorThread(trial_id, agent_id, self.send_queue)
         self.sys_metric_collector_thread.start()
 
-        # TODO: Add data structure to batch timings and then send to SenderThread
-        #       Does this need to be its own thread to flush correctly?
-        # self.timings_batcher = TimingsBatcher()
+        self.timings_batcher = TimingMetricBatcher(trial_id, agent_id)
 
     @property
     def is_enabled(self):
```

Comment:
Since the experiment config field is called "profiling enabled", we should probably rename this variable so it isn't confusing. Maybe `is_active`?